later/lib/database/models/record_model.dart

```diff
@@ -4,10 +4,21 @@ part 'record_model.g.dart';
 
 @collection
 class RecordModel {
+  RecordModel({
+    this.title,
+    this.description,
+    required this.url,
+    required this.lastEditedAt,
+  }) : createdAt = DateTime.now();
```

Comment:
`createdAt` field must be set only when actually creating a _new_ instance of a model.

Imagine the following scenario:
- We're creating an instance of the record model when a user taps the create button. All good, `createdAt` is correct.
- This model is written to the database.
- `isar` restores the model from the database and _creates_ an instance of the record model. But it cannot set the `createdAt` value because it is set automatically. It means that it won't be restored from the database but will be again set to now.